Microsoft.Toolkit.Uwp.UI.Controls/BladeControl/BladeItem.Properties.cs

```diff
@@ -10,6 +10,7 @@
 // THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
 // ******************************************************************
 
+using Windows.Foundation.Metadata;
```

Comment:
Originally BladeId was that class, marked as deprecated, then moved to Blade class. Will remove